uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveRequestTest.php

```diff
@@ -2065,4 +2070,59 @@ public function testLeaveRequestIsDeletedValueCanNotBeSetWhenCreatingALeaveReque
     $leaveRequestRecord = LeaveRequest::findById($leaveRequest->id);
     $this->assertEquals(0, $leaveRequestRecord->is_deleted);
   }
+
+  public function testEmailGetsSentWhenLeaveRequestIsCreated() {
```

Comment:
I wanted to, but I thought it will be a repetition, basically the hook gets triggered, either for an update or create. 
I'll add it though.